manual/manual/library/stdlib.etex

```diff
@@ -110,6 +111,7 @@ be called from C \\
 \item \ahref{libref/Digest.html}{Module \texttt{Digest}: MD5 message digest}
 \item \ahref{libref/Ephemeron.html}{Module \texttt{Ephemeron}: Ephemerons and weak hash table}
 \item \ahref{libref/Filename.html}{Module \texttt{Filename}: operations on file names}
+\item \ahref{libref/Float.html}{Module \texttt{Float}: Floating-point numbers}
 \item \ahref{libref/Format.html}{Module \texttt{Format}: pretty printing}
```

Comment:
Inconsistent capitalization: `: Floating` and `: Ephemerons` but `: operations` and `: pretty-printing`.